pkg/grapicmd/internal/usecase/initialize_project_usecase.go

```diff
@@ -21,7 +23,7 @@ type InitializeProjectUsecase interface {
 }
 
 // NewInitializeProjectUsecase creates a new InitializeProjectUsecase instance.
-func NewInitializeProjectUsecase(ui cli.UI, fs afero.Fs, generator module.ProjectGenerator, excmd excmd.Executor, gexCfg *gex.Config) InitializeProjectUsecase {
+func NewInitializeProjectUsecase(ui cli.UI, fs afero.Fs, generator gencmd.Generator, excmd excmd.Executor, gexCfg *gex.Config) InitializeProjectUsecase {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
line is 153 characters (lll)